docs/usage.rst

```diff
@@ -175,6 +175,15 @@ read the mask before the application runs with hwloc:
 
 .. _pika_stdexec:
 
+``pika-bind`` helper script
+---------------------------
+
+Since versionh ``0.20.0``, the ``pika-bind`` helper script is bundled with ``pika``. ``pika-bind``
+sets the ``PIKA_PROCESS_MASK`` environment variable based on process mask information found before the pika
+runtime is started, and then run the given command. ``pika-bind`` is useful in cases where ``pika`` cannot
+detect the process mask anymore when starting the runtime. This can happen for example if another
+runtime resets the process mask on the main thread before ``pika`` is initialized.
```

Comment:
Since the previous section explains the typical case when `pika-bind` might be used (when used with OpenMP), do you think it'd make sense to write something like this instead:
```suggestion
runtime is started, and then run the given command. ``pika-bind`` is a more convenient alternative to manually setting `PIKA_PROCESS_MASK` when pika is used together with a runtime that may reset the process mask of the main thread, like OpenMP.
```

Reply:
Yes, sounds better.